src/mocks/handlers.ts

```diff
@@ -14,6 +16,29 @@ const containersHandlers: HttpHandler[] = [
   ),
 ];
 
+const namespacesHandlers: HttpHandler[] = [
+  http.get(`${API_URL}/namespaces`, () => {
+    return HttpResponse.json({
+      namespaces: ["namespace1", "namespace2", "namespace3"],
+      count: 3,
+    });
+  }),
+  http.get(`${API_URL}/namespaces/:namespace`, () => {
+    return HttpResponse.json({
+      namespace: "namespace",
+    });
+  }),
+  http.post(`${API_URL}/namespaces`, async ({ request }) => {
+    const requestData = await request.json();
+    console.log(
+      `POST namespaces sent , request:  ${JSON.stringify(requestData)}`
+    );
+    return HttpResponse.json({
+      namespace: "namespace",
+    });
+  }),
+];
+
 const secretsHandlers: HttpHandler[] = [
   http.post(`${API_URL}/:namespace/secrets`, ({ request, params }) => {
     console.log(`POST secret sent, params: ${params} , request: ${request}`);
```

Comment:
Do we need these console logs (I am asking in general) but this specifically seems like it might be a security risk